Tabs Content

azkanurf-ss opened the pull request.

ss-dimasm committed:
fix/address-drm-form

ss-dimasm committed:
Merge branch 'main' into fix/address-drm-form

ss-dimasm committed:
fix: add reusable component

azkanurf-ss committed:
chore:form personal details validation input

azkanurf-ss committed:
chore:get single contact api

azkanurf-ss committed:
merging

azkanurf-ss committed:
Merge branch 'main' of gh_work:abrahamseni/aml-skeleton into azka/form-personal-details

Commit:
Merge branch 'azka/form-personal-details' into hanif/id-form

ss-dimasm committed:
feat: add reusable error message

ss-dimasm committed:
feat: add reusable modal component

ss-dimasm committed:
feat: add generator label field function

azkanurf-ss committed:
fix: merging bug

ss-dimasm committed:
fix: integrate with reusable function and component

ss-dimasm committed:
fix: integrate with reusable function and component

Commit:
feat: add primary and secondary id form

Commit:
Merge branch 'azka/form-personal-details' of github.com:abrahamseni/aml-skeleton into azka/form-personal-details

ss-dimasm committed:
fix: reusable modal component

ss-dimasm committed:
Merge branch 'azka/form-personal-details' into fix/address-drm-form

ss-dimasm committed:
feat: integrate with Backend

ss-dimasm committed:
fix: button handler issue

ss-dimasm committed:
feat: tabs and tabs content switcher

ss-dimasm committed:
feat: add section validator

azkanurf-ss committed:
chore: integrate form personal details

azkanurf-ss committed:
chore:updating useUpdateCntact function

ss-dimasm committed:
merging with 'azka/form-personal-details' into 'fix/address-drm-form'

ss-dimasm committed:
feat: change  ModalDocument component

Commit:
feat: add identity check api

Commit:
Merge branch 'azka/form-personal-details' of github.com:abrahamseni/aml-skeleton into azka/form-personal-details

ss-dimasm committed:
refactor: tabs and switch handler

ss-dimasm committed:
refactor: error message

ss-dimasm committed:
Merge branch 'azka/form-personal-details' into fix/address-drm-form

ss-dimasm committed:
fix: validation schema 'white space'

ss-dimasm committed:
Merge pull request #6 from abrahamseni/fix/address-drm-form

Fix/address drm form

ss-dimasm committed:
fix: page loading issue

ss-dimasm committed:
feat: implement switch tab if success change information

ss-dimasm committed:
feat: add notification message when already communicate with backend

ss-dimasm committed:
refactor: useUpdateContact Hook

azkanurf-ss committed:
chore: define types on personal details form

azkanurf-ss committed:
merging

ss-dimasm committed:
Merge branch 'azka/form-personal-details' into fix/address-drm-form

ss-dimasm committed:
style: adjusting width style and fix label issue

ss-dimasm committed:
fix: unclosed modal

azkanurf-ss committed:
chore: modal status

ss-dimasm committed:
fix: add 'alt' img

ss-dimasm committed:
refactor: remove unecessary utils file

ss-dimasm committed:
refactor: update useUpdateContact hooks

azkanurf-ss committed:
refactor:contact-api

ss-dimasm committed:
Merge branch 'azka/form-personal-details' into fix/address-drm-form

azkanurf-ss committed:
chore: default value status user

ss-dimasm committed:
refactor: adjusting with new useUpdateContact Hook

ss-dimasm committed:
Merge pull request #7 from abrahamseni/fix/address-drm-form

Fix/address drm form

ss-dimasm committed:
refactor: remove unecessary code updateContact hook

ss-dimasm committed:
refactor: remove unecessary code updateContact hook

Commit:
Add primary id test

ss-dimasm committed:
Merge branch 'azka/form-personal-details' into fix/address-drm-form

Commit:
refactor: identity check api

azkanurf-ss committed:
chore: update validation form personal details

ss-dimasm committed:
fix: adjust form styling
refactor: change modal component
feat: add download attribute on 'download' button

ss-dimasm committed:
Merge pull request #9 from abrahamseni/fix/address-drm-form

Fix/address drm form

ss-dimasm committed:
refactor: usage of react-hook-form
fix: next button able to click even with empty field value
style: remove unecessary style attribute

ss-dimasm committed:
Revert "refactor: usage of react-hook-form"

This reverts commit 1f7bc42813fdb5f3107f1825c076dfd57790cf28.

ss-dimasm committed:
refactor: usage of react-hook-form
fix: next button able to click even with empty field value
style: remove unecessary style attribute

Commit:
ui: change validation behavior

ss-dimasm committed:
fix: next button handler not working

ss-dimasm committed:
Merge pull request #10 from abrahamseni/fix/address-drm-form

Fix/address drm form

Commit:
fix: cannot testing component because linaria mock doesn't return correct component

Commit:
refactor: add wait util for testing

Commit:
refactor: change import root to src folder

Commit:
fix linter error

azkanurf-ss committed:
feat: not found id notif

azkanurf-ss committed:
Merge branch 'azka/form-personal-details' of gh_work:abrahamseni/aml-skeleton into azka/form-personal-details

Commit:
feat: add identity check api test

Commit:
Merge branch 'azka/form-personal-details' of github.com:abrahamseni/aml-skeleton into azka/form-personal-details

azkanurf-ss committed:
feat: modal footer

Commit:
feat: add success and error notification to primary and secondary id form

azkanurf-ss committed:
Merge branch 'azka/form-personal-details' of gh_work:abrahamseni/aml-skeleton into azka/form-personal-details

azkanurf-ss committed:
Merge branch 'azka/form-personal-details' of gh_work:abrahamseni/aml-skeleton into azka/form-personal-details

Commit:
feat: change primary and secondary id input labels

Commit:
Merge branch 'azka/form-personal-details' of github.com:abrahamseni/aml-skeleton into azka/form-personal-details

abrahamseni commented on src/components/pages/home-page.tsx:
```diff
@@ -9,32 +9,37 @@ import {
-    </PageContainer>
-  </FlexContainer>
+  <>
+    <SnackProvider>
```
move this to app.tsx

azkanurf-ss committed:
fix: props form footer

azkanurf-ss committed:
Merge branch 'azka/form-personal-details' of gh_work:abrahamseni/aml-skeleton into azka/form-personal-details

azkanurf-ss committed:
feat:button test-id value

ss-dimasm committed:
test: add address information unit testing
test: add drm form unit testing
test add unfinished tab-section unit testing
test: add mock for API response

ss-dimasm committed:
Merge branch 'azka/form-personal-details' into fix/address-drm-form

ss-dimasm committed:
feat: use reusable footer button component

azkanurf-ss committed:
update:form footer

azkanurf-ss committed:
minor fix

abrahamseni commented on src/components/ui/checklist-details-steps/address-information/address-information.tsx:
```diff
@@ -1,86 +1,181 @@
 
+  // turn off disabled attribute, if mutate UpdateContactData state is success
+  // later will do more with optimize way :)
+  React.useLayoutEffect((): void => {
```
why use layouteffect?

ss-dimasm committed:
test: update address information form test

test: update declaration risk management test

refactor: relocate contact mock data

test: add completed-sections test utils function

abrahamseni commented on src/components/ui/checklist-details-steps/personal-details/personal-details.tsx:
```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-confusing-arrow */
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
remove this later

ss-dimasm committed:
Merge pull request #11 from abrahamseni/fix/address-drm-form

Fix/address drm form
